src/test/java/com/serenitydojo/fruitmarket/CatalogTest.java

```diff
@@ -1,30 +1,150 @@
 package com.serenitydojo.fruitmarket;
 
+import com.serenitydojo.*;
+import org.assertj.core.api.Assertions;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.serenitydojo.Fruit.*;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class CatalogTest {
+    Catalog catalog;
+    @Before
+    public void addItemsToCatalog(){
+            catalog = Catalog.withItems(
+            new CatalogItem(Pear),
+            new CatalogItem(Apple),
+            new CatalogItem(Banana),
+            new CatalogItem(Orange)
+        );
+    }
```

Comment:
by convention you usually name the before method `setUp`
but you can give the actions in that the readable name
```suggestion
    @Before
    public void setUp(){
            createNewCatalogWithItems()
        );
    }
    
   private void createNewCatalogWithItems(){
            catalog = Catalog.withItems(
            new CatalogItem(Pear),
            new CatalogItem(Apple),
            new CatalogItem(Banana),
            new CatalogItem(Orange)
        );
    }
    
```